build.sbt

```diff
@@ -310,8 +311,7 @@ lazy val commandProj = (project in file("main-command"))
   .settings(
     testedBaseSettings,
     name := "Command",
-    libraryDependencies ++= Seq(launcherInterface, sjsonNewScalaJson.value, templateResolverApi,
-      jna, jnaPlatform),
+    libraryDependencies ++= Seq(launcherInterface, sjsonNewScalaJson.value, templateResolverApi),
```

Comment:
Why are we dropping JNA? Did we end up not using it anymore in sbt?

Reply:
I am dropping it here so align JNA version with the one introduced by IO.

Reply:
👌